### PR TITLE
[RB-01] Add the calculator code

### PR DESCRIPTION
**Pull request by Rohitsonare**

### CODE REVIEW

File: calculator.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            print(num1, "-", num2, "=", subtract(num1, num2))
+
+        elif choice == '3':
+            print(num1, "*", num2, "=", multiply(num1, num2))
+
+        elif choice == '4':
+            print(num1, "/", num2, "=", divide(num1, num2))
+        
+        # check if user wants another calculation
+        # break the while loop if answer is no
```
Comment (swapnil-jiva) [inline calculator.py:50]:
```suggestion
```

**Review by swapnil-jiva — CHANGES_REQUESTED** [review]
Some minor comms

File: calculator.py (hunk 1 above)
Comment (Rohitsonare) [reply calculator.py:50]: okayy i will change

**Review by swapnil-jiva — APPROVED** [review]
lgtm:)

**Pull request merged**